documentation/src/main/resources/pages/ditto/basic-placeholders.md

```diff
@@ -11,16 +11,78 @@ Placeholders may be used at several places in Ditto where something should be re
+| `{%raw%}{{ thing:name }}{%endraw%}`            | the name (i.e. second part of an ID ) of the related thing                                                                                                                                      |
+### Feature Placeholder
+| Placeholder    | Description                                                                           |
+|----------------|---------------------------------------------------------------------------------------|
+| `{%raw%}{{ feature:id }}{%endraw%}`            | the ID of the feature (only available if the processed signal was related to a feature)                                                                                                         |
+### Policy Placeholder
+| Placeholder                               | Description                                                         |
+|-------------------------------------------|---------------------------------------------------------------------|
+| `{%raw%}{{ policy:id }}{%endraw%}`        | full ID composed of ''namespace'' + '':'' as a separator + ''name'' |
+| `{%raw%}{{ policy:namespace }}{%endraw%}` | the namespace (i.e. first part of an ID) of the related policy      |
+| `{%raw%}{{ policy:name }}{%endraw%}`      | the name (i.e. second part of an ID ) of the related policy         |
+### Policy Entry Placeholder
+| Placeholder                                 | Description                                                         |
+|---------------------------------------------|---------------------------------------------------------------------|
+| `{%raw%}{{ policy-entry:label }}{%endraw%}` |  label of the policy entry in which the token integration subject is injected |
+### Connection Placeholder
+| Placeholder                            | Description                                                                                                                   |
+|----------------------------------------|-------------------------------------------------------------------------------------------------------------------------------|
+| `{%raw%}{{ connection:id }}{%endraw%}` | the ID of the connection which receives/publishes a signal |
+### Header Placeholder
+| Placeholder    | Description                                                                                                                                                                              |
+|----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `{%raw%}{{ header:<header-name> }}{%endraw%}`  | Value of the header with the given name. For incoming signals the headers of the used protocol are used. For outgoing values the Ditto protocol headers are used.(both case-insensitive) |
+### Topic Placeholder
```

Comment:
The "Topic Placeholders" and "Resource Placeholders" are not listed with curly braces.
I guess that is because the sections were moved from "Scope: RQL expressions ..." section - where there must not be curly braces around the placeholders.
For other scopes however there must be the curly braces.

I would suggest to add them everywhere in the tables - and e.g. mention for the new "Scope: RQL expressions ..." section to not include the curly braces.